src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.spring.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
```

Comment:
```suggestion
    @Column(name = "name")           
    private RoleName roleName;
```
be consistent, all your variables in DB have style like: "name" or "name_something"